css/style.css

```diff
@@ -0,0 +1,147 @@
+/* style.css */
+
+* {
+  box-sizing: border-box;
+}
+
+.clearfix:after {
+  content: "";
+  display: table;
+  clear: both;
+}
+
+
+.hero {
+  background-image: url(../images/mountain2.jpg);
+  background-size: cover;
+  color: white;
+  height: calc(100vh - 50px);
+
+
+
+/* Hey future Emileigh: This is flex box. it lets you center things and is amazing!*/
+
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+
+}
+
+.block {
+
+  background-color: black;
+  padding: 40px;
+
+}
+
+
+
+
+a {
+  color: black;
+  font-size: 40px;
+  text-decoration: none;
+  font-family: 'Stalemate', 'Helvetica';
+
+}
+
+h1 {
+  font-size: 50px;
+  font-weight: 400;
+  font-family: 'Raleway', 'Helvetica';
+  /*position: absolute;
+   top: 100px;
+   left: 0;
+   width: 100%;*/
+}
+
+
+h2 {
+  font-size: 70px;
+  font-weight: 400;
+  font-family: 'Stalemate', 'Helvetica';
+  color: #989898;
+  /*position: absolute;
+   top: 150px;
+   left: 0;
+   width: 100%;*/
+
+}
+
+#header {
+  position: relative;
+  width: 100%;
+  text-align: center;
+}
+
+.image {
+  width: 100%;
+  opacity: 0.4;
+  filter: alpha(opacity=40);
+}
+
+.main-text {
+  float: right;
+  width: 60%;
+  padding: 20px;
+  padding-right: 100px;
+  text-align: justify;
+  font-family: 'Raleway', 'Helvetica';
```

Comment:
Good use of a font stack so that Helvetica will show until the webfont loads, or if it doesn't load (instead of Times or whatever the default is)